src/app/datagrid/full/full.ts

```diff
@@ -3,14 +3,15 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import {Component} from "@angular/core";
+import { Component } from "@angular/core";
+
+import { State } from "../../../clarity-angular/data/datagrid/interfaces/state";
+import { FetchResult, Inventory } from "../inventory/inventory";
```

Comment:
Except for the `State` import, everything else looks like a formatting change, no? Does it have to be in this PR? Just in case we revert this PR, the formatting changes will be lost too, no? Shouldn't we create a separate PR for this in case this is needed?